components/core/ConfReader.py

```diff
@@ -0,0 +1,26 @@
+import os
+import inspect
+import json
+import sys
+
+def check_conf_availability():
+    """Checks whether the conf files have been properly set. Exits otherwise"""
+    dlConf = get_conf_reader('dl.conf')
+    emailConf = get_conf_reader('email.conf')
+
+    if any(val == '' for val in dlConf.values()):
+        sys.exit("Please set all the values in dl.conf")
+    if any(val == '' for val in emailConf.values()):
+        sys.exit("Please set all the values in email.conf")
+
+
+def get_conf_reader(confFile):
+    f = open(os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))) + "/" + confFile)
```

Comment:
Still we are getting the conf file from source path right? Lets decide on a conf file dir like ~/.config/bassa on unix and %app_data%/bassa on windows ? And lets move the temp conf to that path upon install or from bassa itself